rockylinux-9.2/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM rockylinux:9.2
+ENV LMOD_VER 8.7
+RUN useradd -ms /bin/bash easybuild
+RUN dnf -y update \
+# enable crb repository
+&& dnf -y install dnf-plugins-core && dnf config-manager --set-enabled crb \
+&& dnf -y install epel-release && dnf -y install python3 python3-pip
+# --allowerasing is required to allow to install curl and remove conflicting curl-minimal which is part of the base image
+# glibc-langpack-en provides locale stuff (for en_US.UTF-8)
+RUN dnf -y --allowerasing install bzip2 curl diffutils file findutils gcc-c++ git glibc-langpack-en gzip make openssl openssl-devel rdma-core-devel patch sudo tar unzip which xz
+# install requirements to build Lmod
+RUN dnf -y install procps-ng lua-filesystem lua-posix lua-devel lua tcl
+# Build Lmod from source
+RUN curl -LO https://sourceforge.net/projects/lmod/files/Lmod-${LMOD_VER}.tar.bz2 \
```

Comment:
@SebastianAchilles I see a very recent Lmod in https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/l/, so can't we install this from EPEL instead?